src/coretoolbox.rs

```diff
@@ -532,10 +532,15 @@ mod entrypoint {
 
         // Allow sudo
         || -> Fallible<()> {
-            let f = std::fs::File::create(format!("/etc/sudoers.d/toolbox-{}", state.username))?;
+            let f_name = format!("/etc/sudoers.d/toolbox-{}", state.username);
+            let f = std::fs::File::create(&f_name)?;
             let mut f = std::io::BufWriter::new(f);
             writeln!(&mut f, "{} ALL=(ALL) NOPASSWD: ALL", state.username)?;
             f.flush()?;
```

Comment:
Let's instead use [set_permissions()](https://doc.rust-lang.org/nightly/std/fs/struct.File.html#method.set_permissions) on the existing file?